GPSCoord[] points
GPSCoord velocity
int64 ID